bm/CMakeLists.txt

```diff
@@ -7,6 +7,10 @@ find_package(Python REQUIRED COMPONENTS Interpreter)
 # thirdparty libs that will be compared with ryml
 
 set(_ed ${CMAKE_CURRENT_BINARY_DIR}/subprojects) # casual ryml extern dir (these projects are not part of ryml and are downloaded and compiled on the fly)
+if(NOT (CMAKE_VERSION VERSION_LESS "4.0.0"))
+    c4_log(STATUS "setting CMAKE_POLICY_VERSION_MINIMUM=3.5 to compile benchmarked libs")
+    set(CMAKE_POLICY_VERSION_MINIMUM 3.5 CACHE STRING "" FORCE)
```

Comment:
This approach is really not recommended. Please adjust the `cmake_minimum_required` with lower and higher bound. That guarantees the policies are scoped for the specific code sections and the lower+higher bound comunicates which CMake versions you have tested in the CI, making it easier to bump this in the future.

Reply:
Thanks for the note. That's already merged; I'll remember your note if this causes any problem. Or feel free to submit a PR.